docs/API.md

```diff
@@ -160,8 +160,9 @@ This parameter is a string that are an IPv4 or IPv6. It's validated with the fol
 
 Basic data type: string
 
-A string of A-Z, a-z and underscores matching the regular expression
-`/^[a-z]{2}(_[A-Z]{2})?$/`.
+A string of A-Z, a-z and underscores matching one of the regular expressions
+`/^[a-z]{2}$/` or `/^[a-z]{2}_[A-Z]{2}$/`. The first format is the preferred
+format, and the second format is *deprecated*.
```

Comment:
I think I would be even more explicit.
```
A string of a-z matching the regular expression `/^[a-z]{2}$/`.
The oldest format matching `/^[a-z]{2}_[A-Z]{2}$/` is *deprecated*.
```

Reply:
I am not sure about "oldest format". I did an update. Is that better?

Reply:
I'm ok with 6f19f65

Reply:
@pnax, can you approve the PR?